### PR TITLE
add option to reopen message

### PR DESCRIPTION
**Pull request by tomashudeckiwi**

### CODE REVIEW

File: phoenix/slackbot/views.py [hunk 1]
```diff
@@ -893,8 +906,11 @@ def resolve(self):
         outage.sales_affected_choice = self.dialog_data.get("sales_affected_choice")
         outage.lost_bookings = lost_bookings
         outage.impact_on_turnover = impact_on_turnover
-        solution.save(modified_by=self.actor)
+
+        if not outage.resolved:
+            outage.resolved = True
```
Comment (aexvir) [inline phoenix/slackbot/views.py:911]: Is this like the default?

Comment (tomashudeckiwi) [reply phoenix/slackbot/views.py:911]: It's there to make sure that when the `Resolve` dialog is being processed the state of `.resolved` will be set to `True`.

Comment (aexvir) [reply phoenix/slackbot/views.py:911]: I see, alright

File: .pre-commit-config.yaml [hunk 2]
```diff
@@ -4,4 +4,4 @@
       name: black-code-formatter
       language: docker_image
       entry: --entrypoint black kiwicom/black:18.9b0
-      types: [python]
+      types: [python]
```
Comment (aexvir) [inline .pre-commit-config.yaml]:
```suggestion
      types: [python]

```

**Comment by aexvir** [comment]
Do you mind splitting the first commit to have meaningful messages? The first commit includes many changes not related to the feature.

I'd squash the commits into a single one and then split the changes to the requirements and the CI to different commits

**Comment by tomashudeckiwi** [comment]
> Do you mind splitting the first commit to have meaningful messages? The first commit includes many changes not related to the feature.
> 
> I'd squash the commits into a single one and then split the changes to the requirements and the CI to different commits

@aexvir  I've divided it into two commits. One for the feature and one that removes coala.

**Review by aexvir — APPROVED** [review]

**Pull request merged**